srcpkgs/linux-asahi/template

```diff
@@ -0,0 +1,325 @@
+# Template file for 'linux-asahi'
+pkgname=linux-asahi
+version=6.11.0p2
+revision=1
+_majver=${version%p*}
+_asahiver=${version%.0p*}
+_asahirel=${version#*.0p}
+_tag=asahi-${_asahiver}-${_asahirel}
+_kernver="${_majver}-${_tag}_${revision}"
```

Comment:
nice, this is much better.

What happens when the kernel version is not `x.yy.0` though? the checks for `.0p` will likely break down in those cases

Reply:
Hmm, you're right. Somehow, they tag `x.y` instead of `x.y.0` but then `x.y.z` for z > 0.

Reply:
I believe this now works.